keyboards/4pplet/eagle_viper_rep/rev_a/config.h

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2021 Stefan Sundin "4pplet" <4pplet@protonmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4444
```

Comment:
This VID is used by `40percentclub` and `4pplet` and both of them has boards in the VIA repo.
Can you come up with a unique one?

Reply:
Thanks for letting me know tho, I was not aware that the same VID was used by them.